mda_openbabel_converter/OpenBabelParser.py

```diff
@@ -3,27 +3,286 @@
+                resnames.append(resid.GetName())
+                chainids.append(resid.GetChainNum()) # is this correct???
+                icodes.append(resid.GetInsertionCode())
+            else:
+                warnings.warn(
+                    f"No residue is defined for atom (ID: {atom.GetIdx()})." 
+                    " Please set with 'MDA SETTING METHOD' if required." # TO DO
+                )
+                resnums.append(None) # TO DO: is this best??
+                resnames.append(None)
+                chainids.append(None)
+                icodes.append(None)
+            
+                 
+            # don't need to check null case, as know assigned to OBMol we're currently parsing
+            # but, NEED TO HANDLE ADDING MULTIPLE SEGIDS/OBMOLS WHEN CONVERTING TO UNIVERSE AND ADDING TOGETHER... (should be ok, check w tests)
+            # segids.append(atom.GetParent())
+            segids.append(0) # need better system!
+
+            # TO DO: may need to create seperate for if SMILES input
+            chirality = None
+            if atom.IsPositiveStereo():
+                chirality = "+"
+            if atom.IsNegativeStereo():
+                chirality = "-"
+            chiralities.append(chirality)
+
+            aromatics.append(atom.IsAromatic()) #boolean
+
+            # altlocs.append()
+            # occupancies.append()
+            # tempfactors.append()
+
+
+        # make Topology attributes
+        attrs = []
+        n_atoms = len(ids)
+
+        if resnums and (len(resnums) != n_atoms): #resnums.__contains__(None):
+            raise ValueError(
+                "ResidueInfo is only partially available in the molecule."
+            )
+
+        # * Attributes always present *
+
+        # Atom attributes
+        for vals, Attr, dtype in (
+            (ids, Atomids, np.int32),
+            (elements, Elements, object),
+            (masses, Masses, np.float32),
+            (aromatics, Aromaticities, bool),
+            (chiralities, RSChirality, 'U1'),
+        ):
+            attrs.append(Attr(np.array(vals, dtype=dtype)))
+
+        # Bonds
+        bonds = []
+        bond_types = []
+        bond_orders = []
+        for bond_idx in range(0, mol.NumBonds()):
+            bond = mol.GetBond(bond_idx)
+            print(bond)
+            bonds.append((bond.GetBeginAtomIdx(), bond.GetEndAtomIdx()))
+            bond_orders.append(bond.GetBondOrder()) # is int, not double. Does this matter?
+
+            # make these a dict instead?
+            OB_BOND_TYPES = [
+            bond.IsAromatic(),
+            bond.IsAmide(),
+            bond.IsPrimaryAmide(),
+            bond.IsSecondaryAmide(),
+            bond.IsEster(),
+            bond.IsCarbonyl(),
+            ]
+
+            MDA_BOND_TYPES = [
+                "aromatic",
+                "amide",
+                "primary amide",
+                "secondary amide",
+                "ester",
+                "carbonyl",
```

Comment:
If you are going to have constants, they need to be defined somewhere else that is programatically accessible, probably best as a subclass of a string Enum. `MDAObabelBondTypes(StrEnum)`